Formula/pulumi.rb

```diff
@@ -2,8 +2,8 @@ class Pulumi < Formula
   desc "Cloud native development platform"
   homepage "https://pulumi.io/"
   url "https://github.com/pulumi/pulumi.git",
-      :tag      => "v0.16.14",
-      :revision => "a627cf817a487d2c9139f4abf936f7e20e141e18"
+      :tag      => "0.16.15",
```

Comment:
I think you've got the wrong tag name - should be `v0.16.15`. https://github.com/pulumi/pulumi/releases/tag/v0.16.15

Reply:
Good catch thanks.